openpype/plugins/publish/extract_otio_audio_tracks.py

```diff
@@ -245,46 +237,80 @@ def create_empty(self, inputs):
             "durationSec": max_duration_sec
         }
 
-    def create_cmd(self, inputs):
+    def mix_audio(self, audio_inputs, audio_temp_fpath):
         """Creating multiple input cmd string
 
         Args:
-            inputs (list): list of input dicts. Order mater.
+            audio_inputs (list): list of input dicts. Order mater.
 
         Returns:
             str: the command body
-
         """
+
+        longest_input = 0
+        for audio_input in audio_inputs:
+            audio_len = audio_input["durationSec"]
+            if audio_len > longest_input:
+                longest_input = audio_len
+
         # create cmd segments
-        _inputs = ""
-        _filters = "-filter_complex \""
-        _channels = ""
-        for index, input in enumerate(inputs):
-            input_format = input.copy()
-            input_format.update({"i": index})
-            input_format["mediaPath"] = path_to_subprocess_arg(
-                input_format["mediaPath"]
+        input_args = []
+        filters = []
+        tag_names = []
+        for index, audio_input in enumerate(audio_inputs):
+            input_args.extend([
```

Comment:
I wonder if this could not be also added into the sidecar file. The similar issue was someone solving here https://stackoverflow.com/questions/64407313/is-it-possible-to-provide-input-files-list-to-ffmpeg-in-a-text-file-instead-of-o